### PR TITLE
History feat

### PR DESCRIPTION
**Pull request by BelideNikhil**
## Features

+ History Feature
   + Save to history
   + Remove Video From History
   + Clear History

### CODE REVIEW

**Review by manojasarsa — APPROVED** [review]
Code looks clean and readable. 
Good to go!

**Review by himadri2110 — APPROVED** [review]
The History feature works as expected.
The code is well structured.
Looks good to be merged!

File: src/Pages/SingleVideo/SingleVideo.jsx [hunk 1]
```diff
@@ -36,13 +36,17 @@ export default function SingleVideo() {
...
+        addToHistoryList({ token, video });
+    }, [videoId, token, video]);
 
     return (
         <div className="main-wrapper">
             <Sidebar />
             {isLoading ? (
-                <Loading />
+                <div>
+                    <h4></h4>
```
Comment (msdeshmukh009) [inline src/Pages/SingleVideo/SingleVideo.jsx]: You can remove this line if not needed.

**Review by msdeshmukh009 — APPROVED** [review]
The code looks clean and the implementation is clean.

File: src/Pages/SingleVideo/SingleVideo.jsx (hunk 1 above)
Comment (BelideNikhil) [reply src/Pages/SingleVideo/SingleVideo.jsx]: done!

**Pull request merged**